### PR TITLE
MAUI-1677 Fixed the effects sub category title issue in SfEffectsView

### PR DESCRIPTION
**Pull request by Kumaraguru1081**
**Description**

I have fixed the effects sub category title issue in SfEffectsView.

### CODE REVIEW

File: maui-toc.html [hunk 1]
```diff
@@ -89,12 +89,12 @@
             <li>
                 <a href="/maui/Effects-View/Effects/Overview">Effects</a>
                 <ul>
-                    <li><a href="/maui/Effects-View/Effects/Highlight"></a></li>
-                    <li><a href="/maui/Effects-View/Effects/Ripple"></a></li>
-                    <li><a href="/maui/Effects-View/Effects/Scale"></a></li>
-                    <li><a href="/maui/Effects-View/Effects/Selection"></a></li>
-                    <li><a href="/maui/Effects-View/Effects/Rotation"></a></li>
-                    <li><a href="/maui/Effects-View/Effects/Combinations"></a></li>
+                    <li><a href="/maui/Effects-View/Effects/Highlight">Highlight</a></li>
```
[the remark this answers is not in the document]
Comment (Kumaraguru1081) [reply maui-toc.html]: Updated.

**Review by Choza-rajan — APPROVED** [review]
Fine

**Comment by Choza-rajan** [comment]
rebuild

**Comment by Choza-rajan** [comment]
rebuild

**Pull request merged**